logic/transaction.js

```diff
@@ -1209,6 +1212,18 @@ class Transaction {
 
 		return transaction;
 	}
+
```

Comment:
Do we really need this method here? I reckon it is redundant. It is just calling ready...

Reply:
The ready method only returns a boolean this method uses the result of that logic to set the correct value. We need it.

Reply:
I still believe if we don't introduce any additional logic to the new function, there is no need to add it. Please, remove it but keep the unit test for `ready` function.

Reply:
Good point, for now that method was sanitising only the ready method so it make sense to just use ready() directly.